arch/arm/configs/tsimx6_minimal_defconfig

```diff
@@ -235,7 +236,6 @@ CONFIG_GPIO_TS4900=y
 CONFIG_POWER_RESET=y
 CONFIG_POWER_RESET_SYSCON=y
 CONFIG_POWER_RESET_SYSCON_POWEROFF=y
-# CONFIG_HWMON is not set
```

Comment:
Unless this was removed because another option selected this, we should keep this in for temperature sensor in CPU.

Reply:
At this point I think all of our platforms need this so I'll add this to our requirements.